fe/fe-core/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -865,6 +862,8 @@ public static void loadJournal(Catalog catalog, JournalEntity journal) {
                     throw e;
                 }
             }
+        } catch (MetaNotFoundException e) {
+            LOG.warn("[INCONSISTENT META] replay failed {}", journal, e);
```

Comment:
add `e.getMessage()` so that it is easier to `grep` in one line.
And some of `journal` may not implement `toString()` method. So I think we can just print op code.

Reply:
I considered printing the original journal content, but gave up because of the complicated deserialization. `JournalEntity` has implemented `toString()` method and has already print the op code. It's better for all the `Writable` objects to implement `toString()` method.
I'll add `e.getMessage()` in one line and keep the exception stack trace.